statistics_server/src/main.rs

```diff
@@ -0,0 +1,66 @@
+extern crate actix;
+extern crate actix_web;
+extern crate futures;
+extern crate althea_types;
+extern crate env_logger;
+extern crate docopt;
+
+#[macro_use]
+extern crate log;
+
+use actix_web::*;
+use futures::{Future};
+
+use docopt::Docopt;
+
+use althea_types::interop::Statistics;
+
+const USAGE: &str = "
+Usage: stats_server --es-url <es_url> --table <table> --bind-url <bind_url>
+Options:
+    --es-url   URL of elasticsearch server to insert data into e.g. 127.0.0.1:1234
+    --bind-url   URL to bind to
+    --table   table name to insert data into
+";
```

Comment:
stats server is separate from the rita, so we would need another settings crate. We will likely need to do that anyways in the future but I thought it would be nice to keep the stats server small for now.

Reply:
That doesn't make anything easier or smaller really, it only defers the work on hooking up a config into the future, or presently into the firmware, where the daemon init script will need to take these config values from somewhere.